lib/reflect.js

```diff
@@ -44,23 +44,19 @@ import wrapAsync from './internal/wrapAsync';
 export default function reflect(fn) {
     var _fn = wrapAsync(fn);
     return initialParams(function reflectOn(args, reflectCallback) {
-        args.push(rest(function callback(err, cbArgs) {
+        args.push(function callback(err, cbArg) {
             if (err) {
                 reflectCallback(null, {
                     error: err
                 });
             } else {
-                var value = null;
-                if (cbArgs.length === 1) {
-                    value = cbArgs[0];
-                } else if (cbArgs.length > 1) {
-                    value = cbArgs;
-                }
```

Comment:
I think I prefer this condition, except removing the `else if` and replacing with an `else`